src/AllBookings/BookingCard.js

```diff
@@ -0,0 +1,281 @@
+// @flow
+
+import * as React from 'react';
+import idx from 'idx';
+import css from 'styled-jsx/css';
+import { graphql, createFragmentContainer } from 'react-relay';
+import { Typography } from '@kiwicom/orbit-components';
+import {
+  ChevronRight,
+  Passengers,
+  FlightReturn,
+  FlightDirect,
+} from '@kiwicom/orbit-components/lib/icons';
+
+import { formatDepartureDate } from '../helpers/dateUtils';
+import bookingTypes from '../common/booking/bookingTypes';
+import bookingStatus from '../common/booking/bookingStatuses';
+import CarrierLogoWrapper from '../SingleBookingPage/bookingItem/CarrierLogoWrapper';
+import type { BookingCard_booking } from './__generated__/BookingCard_booking.graphql';
+
+const styles = css`
+  .card {
+    width: 468px;
+    height: 128px;
+    position: relative;
+    border-radius: 3px;
+    background-color: #ffffff;
+    border: solid 1px #e8edf1;
+    padding: 12px 44px 24px 44px;
+    margin-bottom: 24px;
+  }
+  div.fields {
+    display: flex;
+    justify-content: space-between;
+  }
+  div.logoCarriers {
+    position: absolute;
+    top: 23px;
+    left: 6px;
+  }
+  div.chevron {
+    position: absolute;
+    top: 27px;
+    right: 16px;
+    line-height: 2;
+  }
+  div.label {
+    display: block;
+    margin-bottom: 4px;
+  }
+  div.section {
+    display: block;
+  }
+  p.flight {
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 1.4;
+    color: #46515e;
+    margin-bottom: 12px;
+    margin-left: 2px;
+    display: inline-block;
+  }
+  div.arrowIcon {
+    display: inline-block;
+    margin-left: 8px;
+    margin-right: 8px;
+    vertical-align: top;
+  }
+`;
+
+type Props = {|
+  booking: BookingCard_booking,
+|};
+
+const BookingCard = (props: Props) => {
+  const booking = props.booking;
+  const { passengerCount, type, databaseId } = booking;
+  const status = booking.status && bookingStatus[booking.status];
+
+  let departureDate, origin, destination, IATAOrigin, IATADestination, trip;
+
+  switch (type) {
+    case bookingTypes.ONE_WAY:
+      trip = idx(booking.oneWay, _ => _.trip);
+      departureDate = idx(booking.oneWay, _ => _.trip.departure.time);
+      origin = idx(booking.oneWay, _ => _.trip.departure.airport.city.name);
+      IATAOrigin = idx(
+        booking.oneWay,
+        _ => _.trip.departure.airport.locationId,
+      );
+      destination = idx(booking.oneWay, _ => _.trip.arrival.airport.city.name);
+      IATADestination = idx(
+        booking.oneWay,
+        _ => _.trip.arrival.airport.locationId,
+      );
+      break;
+    case bookingTypes.RETURN:
+      trip = idx(booking.return, _ => _.outbound);
```

Comment:
this creates logo just from part of the trip (same applies to multicity), we need probably to de-deprecate `legs` to make it easier